cdc/model/protocol.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"fmt"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/ticdc/pkg/p2p"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+// This file contains a communication protocol between the Owner and the Processor.
+// FIXME a detailed documentation on the interaction will be added later in a separate file.
+
+// DispatchTableTopic returns a message topic for dispatching a table.
+func DispatchTableTopic(changefeedID ChangeFeedID) p2p.Topic {
+	return fmt.Sprintf("dispatch/%s", changefeedID)
+}
+
+// DispatchTableMessage is the message body for dispatching a table.
+type DispatchTableMessage struct {
+	OwnerRev   int64   `json:"owner-rev"`
+	ID         TableID `json:"id"`
+	IsDelete   bool    `json:"is-delete"`
+}
```

Comment:
Could you add some tests to lock the field names and types? It can prevent unexpected breaking changes, eg. changing type without rename json tag.